include/z64skin.h

```diff
@@ -0,0 +1,85 @@
+#ifndef Z64_SKIN_H
+#define Z64_SKIN_H
+
+#include "z64animation.h"
+
+/**
+ * Holds a compact version of a vertex used in the Skin system
+ * It is used to initialise the Vtx used by an animated limb
+ */
+typedef struct {
+    /* 0x00 */ u16 index;
+    /* 0x02 */ s16 s; // s and t refers to the colloquially known u and v are texture coordinates
```

Comment:
maybe
`// s and t refers to what is commonly known as u and v for texture coordinates`